.github/workflows/go.yml

```diff
@@ -45,3 +46,29 @@ jobs:
 
       - name: Test example/devdata_cli (sub module)
         run: cd ./examples/devdata_cli && go test -race ./...
+
+  integration_test:
+    name: Integration test
+    environment: ${{ github.ref == 'refs/heads/main' && 'production' || '[staging, staging_woop_woop]' }}
```

Comment:
Can we maybe use _dispatch_ setting (will be enabled once merged too main) with array `[staging, production]` as fallback?